libs/domain/site/src/services/resolvers/breadcrumbs.resolver.ts

```diff
@@ -0,0 +1,18 @@
+import { PageMetaResolverService } from '@spryker-oryx/core';
+import { Provider, inject } from '@spryker-oryx/di';
+import { Observable, map } from 'rxjs';
+import { Breadcrumb } from '../../models';
+import { BreadcrumbsResolver, BreadcrumbsResolvers } from '../breadcrumbs';
+
+export class DefaultBreadcrumbsResolver implements BreadcrumbsResolver {
+  constructor(protected pageMetaService = inject(PageMetaResolverService)) {}
+
+  resolve(): Observable<Breadcrumb[]> {
+    return this.pageMetaService.getTitle().pipe(map((text) => [{ text }]));
+  }
+}
+
+export const DefaultBreadcrumbs: Provider = {
+  provide: `${BreadcrumbsResolvers}DEFAULT`,
```

Comment:
can we have one const for this one?

Reply:
lets implement it in the following subtask